### ISSUE TITLE
Unable to load values in Tags in mixed mode at startup

### ISSUE THREAD

**Issue by harmeetsf** [yairEO/tagify#669]
I am loading data from server stored in database. When I put that that data in tagify's value prop then it is not loading it as tags
```
<Tags
	settings={{
		mode: 'mix',
		pattern: /@|#/,
		dropdown : {
			enabled: 0,
			highlightFirst: true
		},
		enforceWhitelist: true,
		duplicates: true
	}}
	tagifyRef={tagifyRef}
	whitelist={whitelistVariables}
	className="input-text"
	value={props.servicePricingFormula || ""}
	onChange={e => (e.persist(), props.onInputChange("servicePricingFormula", e.target.value))}
/>
```

**Comment by yairEO** [comment]
May I see `props.servicePricingFormula`? paste it here

**Comment by harmeetsf** [comment]
**props.servicePricingFormula**
```
 [[{"value":"fola","key":"var5f7835456452261bfc8320c4","class":"private-tag","prefix":"@"}]] 
```

**Output on first load**
![image](https://user-images.githubusercontent.com/67892124/95021030-36d44080-068c-11eb-91b1-da519e09794f.png)

**Now when I take my cursor to end and use **@** to and select a tag then both the tags are rendered...**
![image](https://user-images.githubusercontent.com/67892124/95021089-903c6f80-068c-11eb-87f2-140a9f2331d8.png)

**Also, it gives error in console whenever I add any new tag:**
![image](https://user-images.githubusercontent.com/67892124/95021125-caa60c80-068c-11eb-9762-0b3e4c40fdfd.png)

**Comment by yairEO** [comment]
Are you sure you want to use `mode: 'mix'` ?

**Comment by yairEO** [comment]
I have verified and indeed there is a bug:
https://codesandbox.io/s/tagify-react-issue-669-b598b?file=/src/index.js

I'm working on it

**Comment by harmeetsf** [comment]
Please push this version on NPM

**Comment by yairEO** [comment]
I've pushed to Github and was about to push to NPM and then found horrible bug which i thought would take 5 minutes to solve and it turned out to be more. working on it.

**Comment by yairEO** [comment]
published: `3.19.7`
